uast/uast.go

```diff
@@ -30,6 +30,18 @@ const (
 	Expression
 	Statement
 
+	// Binary comparison operators (usually >, >=, etc)
+	BinaryCompOpEqual
```

Comment:
The equal sign is usually (ab)used on many things in most languages (setting default values, assigment, augmented assigment, start of the method in Nim, etc) I think using OpCompEqual indicates more explicitly that this is the equal sign when used in comparison/test expressions (i. e. == in Cish languages).